.travis.yml

```diff
@@ -1,25 +1,32 @@
-language: python
-services:
+language: bash
+sudo: required
+service:
 - docker
+- docker-compose
 jobs:
   include:
-  - stage: build and test
-    script:
-    - docker build --target test  --tag todo-app:test .
-    - docker run -e SECRET_KEY -e API_KEY -e API_TOKEN todo-app:test tests_e2e
-    - docker run --env-file ./.env.test todo-app:test tests
+    - stage: build and test
+      script:
+      - docker build --target test  --tag todo-app-test:latest .
+      - docker run --env-file .env.test --entrypoint poetry
+        todo-app-test:latest run pytest tests
+      - docker run -e MONGO_CONNECTION_STRING -e MONGODB_COLLECTION_NAME -e SECRET_KEY --entrypoint poetry
```

Comment:
You shouldn't need MONGODB_COLLECTION_NAME because the e2e test sets it to a test one.